zcash_client_sqlite/src/wallet/init/migrations/fix_bad_change_flagging.rs

```diff
@@ -52,7 +51,7 @@ impl RusqliteMigration for Migration {
                      AND sn.from_account_id = {table_prefix}_received_notes.account_id
                      AND {table_prefix}_received_notes.recipient_key_scope = :internal_scope"
                 ),
-                named_params! {":internal_scope": scope_code(Scope::Internal)},
+                named_params! {":internal_scope": KeyScope::INTERNAL.encode()},
```

Comment:
```suggestion
                named_params! {":internal_scope": scope_code(Scope::Internal)},
```
This can't be `TransparentKeyScope::INTERNAL.encode()` because it must work when "transparent-inputs" is not enabled (also, it isn't a transparent key scope).

Reply:
Resolved via 0dcb2b2e74f6d8f0552c6cab01e2e0061150b3ce instead.